pkg/okctl/okctl.go

```diff
@@ -746,6 +746,12 @@ https://www.passwordstore.org/
 		_ = k.Store(keyring.KeyTypeUserPassword, password)
 	}
 
+	//machineAuth := aws.NewAuthMachine( // TODO: make AuthMachine work and add it to 772:aws.New
```

Comment:
Commented code?

Reply:
Hmm. Woops. This is code from the auth part which should not be a part of this PR. I'll remove